include/multipass/platform.h

```diff
@@ -55,6 +55,7 @@ class Platform : public Singleton<Platform>
     virtual QString get_workflows_url_override() const;
     virtual bool is_alias_supported(const std::string& alias, const std::string& remote) const;
     virtual bool is_remote_supported(const std::string& remote) const;
+    virtual bool is_backend_supported(const QString& backend) const; // temporary (?)
```

Comment:
I know I had added that, but I wasn't sure if it wouldn't remain useful after all. I am not sure if we'll ever want the daemon to change drivers "live". Until then, it will only really try to set the driver after a restart, but should still refuse to restart if invalid, so needs a dedicated check.

I didn't dig into the code much though, so just added the interrogation mark. But I can remove if you want?